_preload.lua

```diff
@@ -34,7 +34,14 @@
 			p.clean.directory(prj, prj.name)
 		end
 	}
+	
+--
+-- Add a new flag to put netbeans projects into subdirectories
+--
 
+	p.api.addAllowed("flags", {
+		"nbProjectFolder"
+	})
```

Comment:
I don't think this should be a flag and presume the project name. It should probably be an API that users can specify.